### PR TITLE
Change return chiefs data in order

### PR DESCRIPTION
**Pull request by toomore**
Order members in /api/members

## Pull request type

Please check the type of change your PR introduces:

- [x] Refactoring (no functional changes, no api changes)

## What is the current behavior?

order by user created at.

Issue Number: #78 

## What is the new behavior?

- By manual order in admin page.

## Other information

None

### CODE REVIEW

File: view/api.py [hunk 1]
```diff
@@ -27,7 +27,12 @@ def project_members():
         data['tid'] = team['tid']
 
         data['chiefs'] = []
-        for user in User.get_info(uids=team['chiefs']).values():
+        chiefs_infos = User.get_info(uids=team['chiefs'])
+        for uid in team['chiefs']:
```
Comment (pan93412) [inline view/api.py:31]: 建議加個 comment 稍微說明一下為何要這麼做，不然其實有點難理解為何要這樣寫 🤔

Comment (toomore) [reply view/api.py:31]: `User.get_info` 資料回來是 `dict` 是無序的，這裡會依照後台的設定的排序輸出。

Comment (pan93412) [reply view/api.py:31]: 我等等發個 pr 把這段 comment 寫進去 src 裡面。

Comment (toomore) [reply view/api.py:31]: 這個我等等會直接 merge。

**Pull request merged**